lib/main.dart

```diff
@@ -323,6 +331,7 @@ class WideLayout extends StatelessWidget {
   }
 }
 
+// Carlo
```

Comment:
Good use of a stateful widget so that the UI reflects changes in pending and completed tasks.